packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/__tests__/quanticGeneratedAnswerContent.test.js

```diff
@@ -0,0 +1,120 @@
+// @ts-ignore
+import QuanticGeneratedAnswerContent from 'c/quanticGeneratedAnswerContent';
+import {createElement} from 'lwc';
+
+const mockMarkedUse = jest.fn();
+const mockMarkedParse = jest.fn((text) => text);
+global.marked = {
+  use: mockMarkedUse,
+  parse: mockMarkedParse,
+};
+
+const defaultOptions = {
+  isStreaming: false,
+  answerContentFormat: 'text/plain',
+  answer: '',
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-generated-answer-content', {
+    is: QuanticGeneratedAnswerContent,
+  });
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line @lwc/lwc/no-async-operation
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+describe('c-quantic-generated-answer-content', () => {
+  function cleanup() {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+    jest.clearAllMocks();
+  }
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('text/plain answer content', () => {
```

Comment:
Test idea:
We can also test that the Marked library was not loaded in this situation.

Reply:
We don't actually load the library in the tests, I'm the one mocking it, so we can't test that the library was not loaded.